core/src/main/java/org/mule/runtime/core/api/util/ExceptionUtils.java

```diff
@@ -48,13 +51,32 @@
 import java.util.Optional;
 import java.util.concurrent.Callable;
 
+import org.reactivestreams.Publisher;
 import org.slf4j.Logger;
+import reactor.core.publisher.Mono;
 
 /**
  * Mule exception utilities.
  */
 public class ExceptionUtils {
 
+
+  /**
+   * Null {@link MessagingExceptionHandler} which can we used to configure a {@link MessageProcessorChain} to not handle errors.
+   */
+  public static final MessagingExceptionHandler NULL_ERROR_HANDLER = new MessagingExceptionHandler() {
+
+    @Override
+    public Event handleException(MessagingException exception, Event event) {
+      throw new RuntimeException(exception);
```

Comment:
This matches behaviour in `TemplateOnErrorHandler.java`. Why it throws `RuntimeException` i'm not sure. @afelisatti ?